src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -128,14 +128,21 @@ object ExtractionPipeline {
       linkingEntities = fcReplicate
     )
 
-    // don't need to use files apart from storing them, so we don't assign an output here
+    // don't need to use the following entities apart from storing them, so we don't assign an output here
+
     extractLinkedEntities(
       entityToExtract = EncodeEntity.File,
       matchingField = "@id",
       linkingEntities = SCollection.unionAll(List(experiments, fcExperiments)),
       linkedField = "dataset"
     )
 
+    extractLinkedEntities(
+      entityToExtract = EncodeEntity.AntibodyLot,
+      matchingField = "antibody",
+      linkingEntities = replicates
+    )
```

Comment:
Can you move this up to be right under the `replicates` definition, before the partitioning block?